policykit/policyengine/views.py

```diff
@@ -183,6 +184,8 @@ def initialize_starterkit(request):
         starter_kit = SlackStarterKit.objects.get(name=starterkit_name)
     elif platform == "reddit":
         starter_kit = RedditStarterKit.objects.get(name=starterkit_name)
+    elif platform == "discord":
+        starter_kit = DiscordStarterKit.objects.get(name=starterkit_name)
 
```

Comment:
I missed this the last time around - but can you architect it so that we don't need to be calling integration libraries here?